cmd/export.go

```diff
@@ -104,7 +104,7 @@ func processHashes(sourceDir, exportDir string, hashes map[string]struct{}, dry
 			return err
 		}
 
-		if !!info.IsDir() {
+		if !info.IsDir() {
```

Comment:
This should be reverted or set to `if info.IsDr() {` since we want to check the content type and ignore dirs.

The `!!bool` syntax is useful sometimes, and confusing sometimes.